### ISSUE TITLE
Idea: ISSUE TEMPLATE pointing to the Scope of Work section of the readme

### ISSUE THREAD

**Issue by bumblefudge** [swicg/activitypub-trust-and-safety#22]
(once that gets fleshed out with an "out of scope" section-- glad to come to the meeting where this gets did!)

**Comment by ThisIsMissEm** [comment]
Yeah, probably really want to have several sections in each issue, and to rework existing issues to follow that same pattern.

i.e., what's the issue, what's the current behavior, how is this currently defined in the specification, what is the desired behavior and outcomes, how does this conflict with other approaches / software / specifications?

Would love some help fleshing this out

**Comment by ThisIsMissEm** [comment]
Implementing.
